FastaLenFilter/FastaLenFilter.go

```diff
@@ -0,0 +1,63 @@
+// Copyright ©2017 The bíogo Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Filter sequences that are above a length cut-off
+package main
+
+import (
+	"fmt"
+	"io"
+	"github.com/biogo/biogo/alphabet"
+	"github.com/biogo/biogo/io/seqio/fasta"
+	"github.com/biogo/biogo/seq/linear"
+	"os"
+)
+
+
+var (
+	inf     = flag.String("inf", "test.fna", "input filename")
+	outf    = flag.String("outf", "test_gt2500bp.fna", "output filename")
+	minLen = flag.Int("minLen", 2500, "minimum sequence length cut-off (bp)")
+	help    = flag.Bool("help", false, "help prints this message.")
+)
+
+
+func main() {
+	flag.Parse()
+	if *help {
+		flag.Usage()
+		os.Exit(0)
+	}
+	f, err := os.Open(*inf)
+	  if err != nil {
+		  fmt.Fprintf(os.Stderr, "open input FASTA file: %v.", err)
```

Comment:
`log.Fatalf("failed to open input fasta file: %v", err)` and delete following line.